templates/tfengine/components/cicd/triggers.tf

```diff
@@ -64,6 +69,7 @@ resource "google_cloudbuild_trigger" "validate_{{.name}}" {
   substitutions = {
     _TERRAFORM_ROOT = "{{$terraform_root}}"
     _MANAGED_DIRS = "{{$managed_dirs}}"
+    _WORKER_POOL = "{{$worker_pool}}"
```

Comment:
we should do this for other triggers as well

Reply:
done